### PR TITLE
add introduction and architecture for SKS

### PR DESCRIPTION
**Pull request by rasheedamir**

### CODE REVIEW

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater/stakaterclouddocs:SNAPSHOT-PR-16-1`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater/stakaterclouddocs:SNAPSHOT-PR-16-2`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater/stakaterclouddocs:SNAPSHOT-PR-16-3`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater/stakaterclouddocs:SNAPSHOT-PR-16-4`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater/stakaterclouddocs:SNAPSHOT-PR-16-5`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater/stakaterclouddocs:SNAPSHOT-PR-16-6`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater/stakaterclouddocs:SNAPSHOT-PR-16-7`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater/stakaterclouddocs:SNAPSHOT-PR-16-8`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater/stakaterclouddocs:SNAPSHOT-PR-16-10`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater/stakaterclouddocs:SNAPSHOT-PR-16-12`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater/stakaterclouddocs:SNAPSHOT-PR-16-11`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater/stakaterclouddocs:SNAPSHOT-PR-16-9`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater/stakaterclouddocs:SNAPSHOT-PR-16-13`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater/stakaterclouddocs:SNAPSHOT-PR-16-14`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater/stakaterclouddocs:SNAPSHOT-PR-16-15`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater/stakaterclouddocs:SNAPSHOT-PR-16-16`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater/stakaterclouddocs:SNAPSHOT-PR-16-17`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater/stakaterclouddocs:SNAPSHOT-PR-16-18`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater/stakaterclouddocs:SNAPSHOT-PR-16-19`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater/stakaterclouddocs:SNAPSHOT-PR-16-20`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater/stakaterclouddocs:SNAPSHOT-PR-16-22`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater/stakaterclouddocs:SNAPSHOT-PR-16-21`

File: content/sre/introduction/images/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (kahootali) [inline content/sre/introduction/images/README.md]: why we need this file?

File: .vuepress/config.js [hunk 2]
```diff
@@ -21,6 +21,15 @@ module.exports = {
...
             {
+                title: 'Introduction',
+                children: [
+                    '/content/sre/introduction/introduction',
+                    '/content/sre/introduction/why-sro',
+                    '/content/sre/introduction/sro-overview',
+                    '/content/sre/introduction/sro-features',                  
+                    '/content/sre/introduction/sro-key-differentiators'                    
+                ]
+            },            {
```
Comment (kahootali) [inline .vuepress/config.js]: fix indentation for better readability

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater/stakaterclouddocs:SNAPSHOT-PR-16-23`

**Comment by rasheedamir** [comment]
@kahootali fixed

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater/stakaterclouddocs:SNAPSHOT-PR-16-24`

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater/stakaterclouddocs:SNAPSHOT-PR-16-25`

File: content/sre/introduction/why-sro.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+
+# Why SRO?
+
+- Quicker return on investment > Get up and running quickly
+- Reduce Operational Overhead > Fully managed from infrastructure to day to day operations
+- SRE Support > Enhance security with automated patching and upgrades with zero downtime
+- Empower Developer productivity > Focus on developing apps that add value to the business
+- Flexible consumption based pricing >Optimize costs with ﬂexible pay-as-you-go pricing model (monthly binding)
```
Comment (usamaahmadkhan) [inline content/sre/introduction/why-sro.md]: space after `>`

**Review by usamaahmadkhan — CHANGES_REQUESTED** [review]

**Review by usamaahmadkhan — APPROVED** [review]

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-delivery.stakater.com:443/stakater/stakaterclouddocs:SNAPSHOT-PR-16-26`

**Pull request merged**